test/typescript/complex/fps-spy.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { StagedComponent } from './lib';
+
+export interface FPSSpyProperties {
+  /** Handle FPS change event. */
+  onChange: (fps: number) => void;
+}
+
+export class FPSSpy extends StagedComponent<FPSSpyProperties> {
```

Comment:
what about native FPS counters in browsers? Is it really necessary to make another one for Pixi display only?